### DISCUSSION TITLE
[help needed] How do I attach preinstalled casks to homebrew?

### DISCUSSION THREAD

**Discussion by ghost** [Homebrew/discussions#2648] [Casks] [answered]
So I recently switched to homebrew and some application that I would like to be in installed cask list are not there. I don't want to reinstall those apps because some data like settings can't be saved. Is it possible show homebrew the casks that are already installed?

**Comment by SMillerDev** [comment]
It is not possible to register existing programs with Homebrew.

**Comment by neutric** [comment, marked as answer]
@p0pcat Which apps are behaving that way? Aren't settings normally saved in a separate location like e.g. `~/Library/Preferences` so that you can update easily without losing configuration?

**Reply by ghost** [reply]
yo dude thx, I didn't know that it is possible

**Reply by neutric** [reply]
You're welcome! For the app where it really matters: export the settings, or, if not possible, take screenshots of the relevant settings pages.
